Rubberduck.VBEEditor/Events/IVbeEvents.cs

```diff
@@ -18,5 +18,6 @@ public interface IVBEEvents
         event EventHandler<ReferenceEventArgs> ProjectReferenceAdded;
         event EventHandler<ReferenceEventArgs> ProjectReferenceRemoved;
         event EventHandler EventsTerminated;
+        bool Terminated { get; }
```

Comment:
Am I missing something, or is the diff just screwed up?  I don't see this implemented in VbeEvents.cs.